raiden/constants.py

```diff
@@ -117,13 +106,27 @@ class NetworkType(Enum):
 
 SNAPSHOT_STATE_CHANGES_COUNT = 500
 
-# calculated as of raiden-contracts@d3c30e6d081ac3ed8fbf3f16381889baa3963ea7
-# https://github.com/raiden-network/raiden-contracts/tree/d3c30e6d081ac3ed8fbf3f16381889baa3963ea7
-GAS_REQUIRED_FOR_OPEN_CHANNEL = 109933
-GAS_REQUIRED_FOR_SET_TOTAL_DEPOSIT = 42214
+GAS_LIMIT_UPPER_BOUND = int(0.4 * 3141592)
+GAS_FACTOR = 1.3
```

Comment:
Just out of curiosity, why 1.3?

Reply:
No reason, I just wanted to increase the gas limit a bit to avoid failures if the gas estimation was off by a few percent. 30% is completely arbitrary.